docs/src/components/Navigation/Mobile.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from "react";
+import styled, { css } from "styled-components";
+import { capitalize } from "./helpers";
+import { graphql, useStaticQuery, Link } from "gatsby";
+import { omitNumbers } from "../../utils/common";
+
+const StyledList = styled.ul`
+  padding: 10px 0;
+  & > li {
+    margin-left: 10px;
+    font-weight: 500;
+    font-size: calc(1em - 2px);
+    a {
+      outline: none;
+      &:focus,
+      &:hover {
+        color: ${({ theme }) => theme.orbit.colorTextLinkPrimary};
+      }
+    }
+  }
+`;
+
+const StyledCollectionWrapper = styled.li<{ expanded: boolean }>`
+  ${({ theme, expanded }) => css`
+    padding: 10px 20px;
+    margin: 0 -20px;
+    cursor: pointer;
+    background: #fff;
+    color: ${theme.orbit.colorTextSecondary};
+    outline: 0;
+    span {
+      color: ${expanded && theme.orbit.colorTextPrimary};
+    }
+
+    &:hover {
+      background: ${theme.orbit.paletteCloudLightHover};
+    }
+
+    &:focus {
+      background: ${theme.orbit.paletteCloudLight};
+    }
+  `}
+`;
+
+interface Props {
+  name: string;
+}
+
+export default function Mobile({ name }: Props) {
```

Comment:
For better display name in developer tools.
```suggestion
export default function MobileNavigation({ name }: Props) {
```